### DISCUSSION TITLE
Wallbox Pulsar Plus momentary power or current readings?

### DISCUSSION THREAD

**Discussion by andre-richter** [lbbrhzn/ocpp#449] [Q&A]
Hi all,

[due to network connectivity reasons](https://github.com/lbbrhzn/ocpp/discussions/251#discussioncomment-2383969), I am currently stuck with SW version `5.3.20` on the Pulsar Plus. I never got `5.5.10` running to date.

With `5.3.20`, the only metering sensor values I get are `energy` sensors:

![image](https://user-images.githubusercontent.com/4130005/159095536-b8a5e82c-9f5b-40e9-8530-650a2206330a.png)

Any `current` or `power` sensors are either 0 or Unknown. So I wonder if this is expected, or if `5.5.10`, which I couldn't get to work yet, would offer some more?

Thanks a lot!

**Comment by GrantKeymer** [comment]
> Hi all,
> 
> [due to network connectivity reasons](https://github.com/lbbrhzn/ocpp/discussions/251#discussioncomment-2383969), I am currently stuck with SW version `5.3.20` on the Pulsar Plus. I never got `5.5.10` running to date.
> 
> With `5.3.20`, the only metering sensor values I get are `energy` sensors:
> 
> ![image](https://user-images.githubusercontent.com/4130005/159095536-b8a5e82c-9f5b-40e9-8530-650a2206330a.png)
> 
> Any `current` or `power` sensors are either 0 or Unknown. So I wonder if this is expected, or if `5.5.10`, which I couldn't get to work yet, would offer some more?
> 
> Thanks a lot!

Have a look at the bottom of this page and you will see the useful Metrics (sensors) which work for me with V5.5.10...

https://github.com/lbbrhzn/ocpp/wiki/user-guide

**Reply by andre-richter** [reply]
Thanks, good to know `Power Active Import` works on your side.

It is constant `0` on my side unfortunately.

**Reply by miqm** [reply]
For me I also have Power Active Import set to 0. However I've a 3-phase model. could this be the reason? I assume to calculate proper power, It'd need voltage and current. but it does calculate energy properly 🤔

**Comment by Strixx76** [comment]
Works for me as well.

The reason I change from the "official" Wallbox integration using the myWallbox cloud service was that the momentary power readings wasn't working correct. It only showed the power in the beginning of each session and then it went down to 0. I saw the same behavior in the Wallbox portal.

I now have both of them running, and I see that it still behaves the same in the "official" integration, but this integration (OCPP) works perfect!

**Reply by GrantKeymer** [reply]
Yes, I've found the same thing with the power readings dropping to 0.  The Wallbox app is poorly written and the Wallbox portal isn't much better.  The only thing I use them for is Session Data, with the monthly graphs being quite handy.
